Code/Mantid/Framework/Algorithms/test/ConvertToMatrixWorkspaceTest.h

```diff
@@ -39,19 +39,23 @@ class ConvertToMatrixWorkspaceTest : public CxxTest::TestSuite
 
   void testExec_2D_to_2D()
   {
+	Mantid::Algorithms::ConvertToMatrixWorkspace cloner;
     if ( !cloner.isInitialized() ) cloner.initialize();
```

Comment:
Just call cloner.initialize(), you know from the line above that it's not going to be initialized yet. No need to check beforehand